script/Arcueid/Day 5/05_00_ARC05_8.txt

```diff
@@ -462,27 +462,27 @@ It's as though he were grinning in agreement with Arcueid's provocation.
 [sha:b8aaf187460b84ead817cb5a6b19a3c96c926449]{
 -- Page 548, Offset 10071.
 -- 「最も強きものとは千変にして<偏|かたよ>らぬもの。
-"The strongest ones are those who continue to adapt and do not remain locked in their ways.
+"Those that strong be... retain unfaltering through vicissitudes do.
 }
 [sha:52f1281e1a8e19a264a2c690652cc29cab9f9ace]{
 -- Page 548, Offset 10072.
 -- ……貴様はまさにソレだ。そこまで衰退していながら、まだおれを上回る余力がある。
-...You are exactly that. Even diminished as you are, your remaining power still surpasses my own.
+And that is truly what you are... Though diminished your power may be... above mine it is nonetheless.
 }
 [sha:0cf50eb40e3ac194055940c470a381326109bc97]{
 -- Page 548, Offset 10073.
 -- あらゆる呪い、あらゆる状況に対応する原初の<一|いち>―――並の死徒ではかなうまい」
-A primordial one that deals with every curse and every situation――― No ordinary Dead Apostle can possibly compare."
+One who can with all curses deal; through all situations adapt: The Original One. An ordinary Dead Apostle stands no chance...
```

Comment:
Against one who can

Reply:
It's poetry, mom.
For context, "The Original One" is Arcueid, and she's the one who can "deal with all curses and situations", then Vlov goes, "An ordinary Dead Apostle stands no chance [against you]"

Reply:
well I also misinterpreted the structure of this sentence entirely so just disregard